### DISCUSSION TITLE
Should I add translations for Slovak?

### DISCUSSION THREAD

**Discussion by PesBandi** [alefragnani/vscode-bookmarks#712] [Q&A] [answered]
Hello, I noticed that the latest release got a new language and I thought I'd help translate too. I started translating to Slovak, but then realized that VS Code doesn't support Slovak, so the translations would never be used. Should I translate it anyway and the files will just sit there waiting for a time when VS Code gets Slovak, or should I stop?

**Comment by alefragnani** [comment, marked as answer]
Hi @PesBandi ,

First of all, thank you for your intention to contribute. It is always welcome. 

But, I think you could, at first, contribute to VS Code itself, using the documentation provided here https://code.visualstudio.com/docs/getstarted/locales#_can-i-contribute-to-a-language-packs-translations. I’m not entirely sure how it work , the time and effort it will take to this to be available, but unless a language pack is available, the extension won’t translate.

I did contribute to the Brazilian Portuguese myself in the past, but it was using a different platform (Transiflex) back then. It seems today it is only the `vscode-loc` repo, with every string available.

Hope this helps
